src/shell/components/withAi/index.tsx

```diff
@@ -111,114 +111,110 @@ export const withAI = (WrappedComponent: ComponentType) =>
       }
     };
 
-    if (isEnabled) {
```

Comment:
I left it in case we opt to lock this again to certain instances and just re-add the variable without having to rewrite the logic again. But I can remove it if it's preferred.